server/src/test/java/keywhiz/KeywhizTestRunner.java

```diff
@@ -104,8 +67,9 @@ static Injector createInjector() {
       jooqContext.truncate(Secrets.SECRETS).execute();
     } catch(DataAccessException e) {}
 
-    Object object = injector.getInstance(getTestClass().getJavaClass());
-    MockitoAnnotations.initMocks(object);
-    return object;
+    Object test = getTestClass().getJavaClass().getDeclaredConstructor().newInstance();
+    injector.injectMembers(test);
+    MockitoAnnotations.initMocks(test);
+    return test;
```

Comment:
Now that we have strict bindings, Guice no longer allows us to request instances of our test classes because their default constructors aren't marked with @Inject. Rather than modify every test class, we just modify how we create and inject them.